llm_apis/ollama_client.py

```diff
@@ -24,6 +26,7 @@ def stream_completion(self, messages, model, **kwargs):
         Args:
             messages (list): List of messages used as context or prompt.
             model (str): Model identifier for text generation.
+            keep_alive (number/str): The duration that models stay loaded in memory (default "5m"). Set to "-1" to keep models loaded indefinitely
```

Comment:
```suggestion
```
This line should be removed since stream_completion() doesn't have a keep_alive parameter.

Reply:
My bad. Removed.